common4j/src/main/com/microsoft/identity/common/java/commands/parameters/BrokerInteractiveTokenCommandParameters.java

```diff
@@ -77,6 +77,9 @@ public class BrokerInteractiveTokenCommandParameters extends InteractiveTokenCom
     @Expose
     private final String homeTenantId;
 
+    // Parameter representing if this broker request is an ATv2 request
+    private final boolean isATv2Request;
```

Comment:
>isATv2Request [](http://example.com/codeflow?start=26&length=13)

how and where do we populate this?

Reply:
This will come in BrokerRequest, the change for that is in the API-side PR (On the way)